components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5629,7 +5629,7 @@ def contains_unsafe_arg(arguments):
 
         if methods:
             self.cgRoot = CGWrapper(CGIndenter(CGList(methods, "")),
-                                    pre="pub trait %sMethods {\n" % descriptor.interface.identifier.name,
+                                    pre="#[must_root] pub trait %sMethods {\n" % descriptor.interface.identifier.name,
```

Comment:
Why is this change required?

Reply:
Because of [this](https://github.com/servo/servo/pull/20264#issuecomment-394314007). We want traits to be `#[must_root]`.